packages/instrumentations/src/index.ts

```diff
@@ -56,14 +56,10 @@ import { SocketIoInstrumentation } from "@opentelemetry/instrumentation-socket.i
 import { TediousInstrumentation } from "@opentelemetry/instrumentation-tedious"
 import { WinstonInstrumentation } from "@opentelemetry/instrumentation-winston"
 import {
-  awsBeanstalkDetector,
   awsEc2Detector,
-  awsEcsDetector,
-  awsEksDetector,
   awsLambdaDetector,
 } from "@opentelemetry/resource-detector-aws"
 import { containerDetector } from "@opentelemetry/resource-detector-container"
-import { gcpDetector } from "@opentelemetry/resource-detector-gcp"
```

Comment:
Some of these detectors were spamming warnings, better just add them back if someone ever requests it.